src/swagger/configs.ts

```diff
@@ -0,0 +1,29 @@
+import swaggerJsdoc, { OAS3Definition } from "swagger-jsdoc";
+
+export const port = 3000;
```

Comment:
Better to move it to separate `configs.ts` file in `src` folder not related to swagger and import here.